requirements.txt

```diff
@@ -1,2 +1,3 @@
 wheel == 0.31.1
 pyelftools >=0.24
+https://lief-project.github.io/packages/lief/lief-0.10.0.dev0-cp37-cp37m-manylinux1_x86_64.whl
```

Comment:
`lief` has been published in pypi. Why don't you install it from there?

Reply:
I am, but I haven't yet pushed the latest changes. I just did now. Not that this is still a work in progress.